### PR TITLE
Prepare release for User and Error pages.

### PR DESCRIPTION
**Pull request by farahTW**
This PR make changes to the User  pages to prepare for release
It does the following

It removes the get_layout method from user controller.
It replaces the layout method in user controller with "design_system".
It removes bootstrap implementation legacy files.
It removes bootstrap implementation from feature tests.

**Note:**  No changes required for Error pages for this release as these are implemented in design system layout.

**Trello:**
https://trello.com/c/tIUQ9pDb/578-prepare-release-for-user-and-error-pages

### CODE REVIEW

File: app/controllers/admin/users_controller.rb
Comment (minhngocd) [inline app/controllers/admin/users_controller.rb:17]: I believe you might still need a return here otherwise it will render the edit page? 
Can try it out?

**Review by minhngocd — COMMENTED** [review]
Are the Error pages also part of this release? or does it require something different?

File: features/step_definitions/user_steps.rb
Comment (davidtrussler) [inline features/step_definitions/user_steps.rb:38]: I haven't tried running this but I would have expected this line to still be necessary since it's outside the `if` clause.

Comment (farahTW) [reply features/step_definitions/user_steps.rb:38]: Hi David as part of new implementation we do have user setting link anymore.We will check it again though.

File: app/controllers/admin/users_controller.rb
Comment (farahTW) [reply app/controllers/admin/users_controller.rb:17]: Hi Minno,
Ideally Edit link should not appear if user is not having GDS editor permission.
I will check again

Comment (minhngocd) [reply app/controllers/admin/users_controller.rb:17]: Hey Farah - the link would not appear, but if you visit the URL, i think the page will still render unless we have this return line

File: features/step_definitions/user_steps.rb
Comment (mtaylorgds) [reply features/step_definitions/user_steps.rb:38]: @davidtrussler The line above is already visiting the user settings page, I believe? So clicking the link takes them to the page they're already on.

If the desire is to test the link then maybe we should be visiting another page and following the link from there, otherwise I don't think we need the link…thoughts?

Comment (davidtrussler) [reply features/step_definitions/user_steps.rb:38]: @farahTW / @mtaylorgds Yes, that's fine. Probably the link shouldn't have been there still but anyway LGTM.

**Review by davidtrussler — APPROVED** [review]
LGTM, but there is still an outstanding discussion to be resolved before merging.

**Comment by minhngocd** [comment]
return renders resolved, LGTM

**Comment by farahTW** [comment]
Merging the PR after all discussions resolved with @minhngocd  and @davidtrussler  as a part of  code review.

**Pull request merged**
